library/src/interactor_impl.cxx

```diff
@@ -172,228 +176,142 @@ class interactor_impl::internals
-        render = true;
+        self->Interactor.triggerCommand("toggle render.raytracing.enable");
         break;
       case 'D':
-        self->Options.render.raytracing.denoise = !self->Options.render.raytracing.denoise;
-        render = true;
+        self->Interactor.triggerCommand("toggle render.raytracing.denoise");
         break;
       case 'V':
-        self->Options.model.volume.enable = !self->Options.model.volume.enable;
-        render = true;
-        printColoring = true;
+        self->Interactor.triggerCommand("toggle_volume_rendering");
         break;
       case 'I':
-        self->Options.model.volume.inverse = !self->Options.model.volume.inverse;
-        render = true;
+        self->Interactor.triggerCommand("toggle model.volume.inverse");
         break;
       case 'O':
-        self->Options.model.point_sprites.enable = !self->Options.model.point_sprites.enable;
-        render = true;
+        self->Interactor.triggerCommand("toggle model.point_sprites.enable");
         break;
       case 'U':
-        self->Options.render.background.blur = !self->Options.render.background.blur;
-        render = true;
+        self->Interactor.triggerCommand("toggle render.background.blur");
         break;
       case 'K':
-        self->Options.interactor.trackball = !self->Options.interactor.trackball;
-        render = true;
+        self->Interactor.triggerCommand("toggle interactor.trackball");
         break;
       case 'F':
-        self->Options.render.hdri.ambient = !self->Options.render.hdri.ambient;
-        render = true;
+        self->Interactor.triggerCommand("toggle render.hdri.ambient");
         break;
       case 'J':
-        self->Options.render.background.skybox = !self->Options.render.background.skybox;
-        render = true;
+        self->Interactor.triggerCommand("toggle render.background.skybox");
         break;
       case 'L':
       {
-        const double intensity = self->Options.render.light.intensity;
-        const bool down = rwi->GetShiftKey();
-
-        /* `ref < x` is equivalent to:
-         * - `intensity <= x` when going down
-         * - `intensity < x` when going up */
-        const double ref = down ? intensity - 1e-6 : intensity;
-        // clang-format off
-        /* offset in percentage points */
-        const int offsetPp = ref < .5 ?  1
-                           : ref <  1 ?  2
-                           : ref <  5 ?  5
-                           : ref < 10 ? 10
-                           :            25;
-        // clang-format on
-        /* new intensity in percents */
-        const int newIntensityPct = std::lround(intensity * 100) + (down ? -offsetPp : +offsetPp);
-
-        self->Options.render.light.intensity = std::max(newIntensityPct, 0) / 100.0;
-        render = true;
+        std::string command = "increase_light_intensity";
+        command += rwi->GetShiftKey() ? " negative" : "";
```

Comment:
Not super fan of this.  
Should we have `increase_light_intensity` and `decrease_light_intensity`?  
Or maybe `light_intensity increase/decrease`?

Reply:
>  Or maybe light_intensity increase/decrease?

That would break the `action` logic.

> increase_light_intensity and decrease_light_intensity

Ill do that.